src/walkway.js

```diff
@@ -324,8 +328,10 @@
   function getPolylineLength(polyline) {
     var dist = 0;
     var x1, x2, y1, y2;
+    var i;
+    var points = polyline.points.numberOfItems;
 
-    for(var i = 1; i < polyline.points.numberOfItems; i++){
+    for (i = 1; i < points; i++){
       x1 = polyline.points.getItem(i - 1) .x;
```

Comment:
It is valid, but it's not good practice. Can you also remove those spaces in this pr please? 
<img width="115" alt="screen shot 2016-04-18 at 8 46 52 am" src="https://cloud.githubusercontent.com/assets/3777511/14610291/26f540a6-0542-11e6-91f5-c26dff406195.png">